app/assets/javascripts/orders.coffee

```diff
@@ -81,6 +88,9 @@ addListeners = (element) ->
     populateItems @value, items
 
 $(document).on 'page:change', ->
+  return unless $("body.orders.index").length > 0
   element = findLastCategory()
-  populateCategories element
+  # populateCategories element
   addListeners element
+
+  $('.data-table').DataTable()
```

Comment:
This seems to be repeated unnecessarily in a bunch of files... how about just have 1 file that is intended to initialize all data tables? Like maybe in `data-tables-init.coffee`:
```coffeescript
$(document).on "page:change" ->
  $(".data-table").DataTable()
```

This would then initialize any data tables that exist, but if none exist, it wouldn't do anything since the selector would result in nothing.